src/app/share-preview/components/share-preview/share-preview.component.ts

```diff
@@ -194,7 +194,7 @@ export class SharePreviewComponent implements OnInit, OnDestroy {
       this.hasRequested = !!this.sharePreviewVO.ShareVO;
       this.hasAccess = this.hasRequested && this.sharePreviewVO.ShareVO.status.includes('ok');
       if (this.sharePreviewVO?.autoApproveToggle === 1) {
-        this.requestAccessText = 'Accept Share';
+        this.isAutoApprove = true;
```

Comment:
It looks like `this.sharePreviewVO` is initialized upon construction; can we also initialize `this.isAutoApprove` at the same time? I confess I do not fully know how object initialization works, but it would be great if we could just write

```ts
  public isAutoApprove: boolean = (this.sharePreviewVO?.autoApproveToggle === 1);
```